addons/spreadsheet_dashboard/security/security.xml

```diff
@@ -1,4 +1,12 @@
 <odoo>
+
+    <record id="ir_rule_spreadsheet_dashboard_group" model="ir.rule">
+        <field name="name">Spreadsheet dashboard group</field>
+        <field name="model_id" ref="model_spreadsheet_dashboard_group"/>
+        <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+        <field name="domain_force">[('id', 'in', user.groups_id.ids)]</field>
```

Comment:
You are comparing apples to oranges.
ids from `spreadsheet.dashboard.group` to ids of `res.groups`, it can't work

Reply:
my bad, its a mistake from my side, will correct it and update a better rule